src/game/objects/har.c

```diff
@@ -1381,11 +1381,18 @@ int har_collide_with_har(object *obj_a, object *obj_b, int loop) {
             // prevent next move from being interrupted
             a->executing_move = 1;
 
+            int retval = move->category == CAT_CLOSE ? 1 : 0;
+
+            if(loop == 0) {
+                // recurse so jaguar's overhead throw grabs enemy on first tick of anim 22
```

Comment:
So if the move chains to another move, check collision on that immediately?

Reply:
Yeah, that's the thought